pynxos/device.py

```diff
@@ -16,12 +16,13 @@ class RebootSignal(NXOSError):
 
 
 class Device(object):
-    def __init__(self, host, username, password, transport=u'http', port=None, timeout=30,
-                 verify=True):
+    def __init__(self, host, username, password, transport=u'http', encoding=u'xml',
+                 port=None, timeout=30, verify=True):
```

Comment:
We should have the encoding default to 'json-rpc' such that pynxos behaves identical to its current behavior for other code that integrates to this library (including the NAPALM library).